web/cobrands/zurich/layout.scss

```diff
@@ -187,6 +187,20 @@ body.mappage {
     #zurich-footer {
         display: none;
     }
+
+    #map_sidebar,
+    #map_box {
+      top: 10em;
+    }
+
+    #zurich-main-nav {
```

Comment:
This means Zurich lose a navbar's worth of map height. Would it work if the nav-wrapper-2 height was kept at 117px, #map_box got 117px top instead of 10em, and zurich-main-nav had top of 117px, no right or left, width of 29em? That seems to better match the existing.